src/Adapters/EventSource.Netstandard13/EventSourceTelemetryModule.cs

```diff
@@ -219,5 +250,50 @@ private void EnableAsNecessary(EventSource eventSource)
                 }
             }
         }
+
+        /// <summary>
+        /// Returns true when eventSource satisfied the rule; false otherwise. Returns false when either is null.
+        /// </summary>
+        /// <param name="eventSource">The target event source.</param>
+        /// <param name="rule">The naming rule to be used for matching.</param>
+        private bool IsEventSourceNameMatch(EventSource eventSource, EventSourceNamingMatchRuleBase rule)
+        {
+            if (string.IsNullOrEmpty(rule?.Name) || string.IsNullOrEmpty(eventSource?.Name))
+            {
+                return false;
+            }
+
+            if (!rule.PrefixMatch)
+            {
+                return string.Equals(eventSource.Name, rule.Name, StringComparison.Ordinal);
+            }
+            else
+            {
+                return eventSource.Name.StartsWith(rule.Name, StringComparison.Ordinal);
+            }
+        }
+
+        /// <summary>
+        /// When the event comes from a EventSource that matches the rule of DisabledSource, it should be dropped, unless the event source name matches one of the 
+        /// name exactly in the enabling rule.
+        /// </summary>
+        /// <param name="eventData">The event data to test.</param>
+        private bool IsDroppingEvent(EventWrittenEventArgs eventData)
+        {
+            string eventSourceName = eventData?.EventSource?.Name;
+            if (string.IsNullOrEmpty(eventSourceName))
+            {
+                EventSourceListenerEventSource.Log.NullReference(nameof(EventSourceListener.EventSourceTelemetryModule), nameof(eventSourceName), $"{nameof(eventSourceName)} can't be null.");
```

Comment:
Just return false. I think it is possible to have an EventSource with just a GUID

Reply:
I knew I forgot something and that is exactly the thing. Thanks for catching that. It will be fixed in the next iteration.